src/zulqer/groupPeople/data.ts

```diff
@@ -0,0 +1,159 @@
+const data = [
```

Comment:
@nirajunify we have a file here to store the test data and is useful only for test purposes. Please advise on where to keep and the filename.

Reply:
Since this data is problem-specific, and the current solution requires it immediately - keep the data file in the same directory as the solution. However, datafile should be a JSON file instead of ts.